zebedee-cms/src/main/java/com/github/onsdigital/zebedee/util/slack/PostMessageAttachment.java

```diff
@@ -0,0 +1,42 @@
+package com.github.onsdigital.zebedee.util.slack;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PostMessageAttachment {
+    public enum Color {
+        Good,
+        Warning,
+        Danger
+    }
+
+    public String pretext;
```

Comment:
Unless there is a specific reason class vars should be private and you should use accessor methods to get/set them.